keyboards/test/ja-Latn-test.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboardTest SYSTEM "../dtd/ldmlKeyboardTest.dtd">
+<keyboardTest conformsTo="techpreview">
+	<info keyboard="ja-Latn.xml" author="Team Keyboard" name="ja-Latn-test" />
+	<repertoire name="latn-repertoire"
+		chars="[a-z A-Z 0-9 !\u0022#$%\u0026\[\]\{\}=\-|¥~\^_\u0020\u003c>,./?`@\+\*]" type="simple" />
```

Comment:
Unsure about spaces between `a-z` and `A-Z` and `0-9` etc here.

Reply:
it's just for possible clarity.  Whitespace is not significant here.